relayer/relays/ethereum/ethereum-listener.go

```diff
@@ -20,25 +22,24 @@ import (
 	"github.com/snowfork/snowbridge/relayer/chain"
 	"github.com/snowfork/snowbridge/relayer/chain/ethereum"
 	"github.com/snowfork/snowbridge/relayer/contracts/basic"
-	"github.com/snowfork/snowbridge/relayer/contracts/incentivized"
 	"github.com/snowfork/snowbridge/relayer/relays/ethereum/syncer"
 
 	log "github.com/sirupsen/logrus"
 )
 
 // EthereumListener streams the Ethereum blockchain for application events
 type EthereumListener struct {
-	ethashDataDir               string
-	ethashCacheDir              string
-	config                      *SourceConfig
-	conn                        *ethereum.Connection
-	basicOutboundChannel        *basic.BasicOutboundChannel
-	incentivizedOutboundChannel *incentivized.IncentivizedOutboundChannel
-	mapping                     map[common.Address]string
-	payloads                    chan ParachainPayload
-	headerSyncer                *syncer.Syncer
-	initBlockHeight             uint64
-	descendantsUntilFinal       uint64
+	ethashDataDir        string
+	ethashCacheDir       string
+	config               *SourceConfig
+	conn                 *ethereum.Connection
+	basicOutboundChannel *basic.BasicOutboundChannel
+	// TODO: remove mapping, it only maps the basic channel address to the string "BasicInboundChannel.submit"
+	mapping               map[common.Address]string
```

Comment:
Prefer to remove/refactor in this PR.

Reply:
Definitely. I had done it, just hadn't pushed until now.